app/controllers/projects/settings/life_cycle_steps_controller.rb

```diff
@@ -26,33 +26,33 @@
 # See COPYRIGHT and LICENSE files for more details.
 # ++
 
-class Projects::Settings::LifeCyclesController < Projects::SettingsController
+class Projects::Settings::LifeCycleStepsController < Projects::SettingsController
   include OpTurbo::ComponentStream
 
   before_action :deny_access_on_feature_flag
 
-  before_action :load_life_cycle_definitions, only: %i[show enable_all disable_all]
+  before_action :load_life_cycle_definitions, only: %i[index enable_all disable_all]
 
-  menu_item :settings_life_cycles
+  menu_item :settings_life_cycle_steps
 
-  def show; end
+  def index; end
 
   def toggle
-    definition = Project::LifeCycleStepDefinition.where(id: params[:definition_id])
+    definition = Project::LifeCycleStepDefinition.where(id: params[:id])
```

Comment:
Very much nitpick - value is still an array, so there may be a bit of confusion why relation is assigned to variable with singular name

Reply:
The variable name is correct in being singular since only a single element will ever be returned. So I find that appropriate. One could switch to using a `find` and wrap that into an array explicitly but that I find too cumbersome.